locations/spiders/timhortons.py

```diff
@@ -2,52 +2,67 @@
 import re
 from locations.items import GeojsonPointItem
 
-class TimhortonsSpider(scrapy.Spider):
 
+class TimhortonsSpider(scrapy.Spider):
     name = "timhortons"
     allowed_domains = ["locations.timhortons.com"]
+    custom_settings = {
+        'DUPEFILTER_CLASS': 'scrapy.dupefilters.BaseDupeFilter',
```

Comment:
Why did you need to add this?

Reply:
I had to hit parse_city_stores sometimes twice (in def parse_city:) depending if there were multiple stores for one URL so I had to do a try except to figure out if the url contained 1 store or multiple.  Scrapy by default does not allow you to send a request on the same URL twice. The original scraper assumed single store/multiple stores based on regex of the URL but that was incorrect.